<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicUserDialog</class>
 <widget class="QWidget" name="MusicUserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="topTitleBar">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>331</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:black</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>4</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>用户登录</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>7</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>29</y>
     <width>330</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="one">
    <widget class="QLabel" name="usernameLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>23</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>帐号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="passwdLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>62</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="userComboBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>170</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwLineEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>170</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="forgotPwdButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>62</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>忘记密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="registerButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>23</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>注册帐号</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="rememberPwd">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>90</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>记住密码</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="automaticLogon">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>90</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>自动登录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>100</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QLabel" name="otherLogin">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>145</y>
       <width>115</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>使用其他账号登录：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="useTencentLogin">
     <property name="geometry">
      <rect>
       <x>152</x>
       <y>148</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/usermanager/lb_qq)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="useRenrenLogin">
     <property name="geometry">
      <rect>
       <x>174</x>
       <y>148</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/usermanager/lb_renren)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="useWechatLogin">
     <property name="geometry">
      <rect>
       <x>196</x>
       <y>148</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/usermanager/lb_weixing)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="useSinaLogin">
     <property name="geometry">
      <rect>
       <x>218</x>
       <y>148</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/usermanager/lb_xinlang)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="two">
    <widget class="QLabel" name="usernameRegisterLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>25</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>注册帐号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="mailLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>77</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>邮箱地址：</string>
     </property>
    </widget>
    <widget class="MusicUserLineEdit" name="registerMailLine">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>75</y>
       <width>190</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="MusicUserLineEdit" name="registerUserLine">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>190</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit::hover{ border: 2px solid #AAAAFF;}</string>
     </property>
    </widget>
    <widget class="QLabel" name="registerUserLineR">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>50</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>4-20个字母、数字和符号组成</string>
     </property>
    </widget>
    <widget class="QLabel" name="registerMailLineR">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>105</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>请填写正确的邮箱地址</string>
     </property>
    </widget>
    <widget class="QLabel" name="registerPwdLineR">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>160</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>6-16个字母、数字和符号组成</string>
     </property>
    </widget>
    <widget class="QLabel" name="registerPwdCLineR">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>215</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>6-16个字母、数字和符号组成</string>
     </property>
    </widget>
    <widget class="MusicUserLineEdit" name="registerPwdLine">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>190</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="MusicUserLineEdit" name="registerPwdCLine">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>185</y>
       <width>190</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="passwdLable">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>132</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>设置密码：</string>
     </property>
    </widget>
    <widget class="QLabel" name="passwdCLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>188</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>确认密码：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirmButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>250</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>250</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="agreementCheckBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>265</y>
       <width>125</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>我已阅读并接受</string>
     </property>
    </widget>
    <widget class="QPushButton" name="agreementButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>240</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>网站协议</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRight">
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>22</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image:url(:/tiny/lb_right)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelRigh_2">
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>77</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image:url(:/tiny/lb_right)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelRigh_3">
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>132</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image:url(:/tiny/lb_right)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelRigh_4">
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>187</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image:url(:/tiny/lb_right)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="three">
    <widget class="QLabel" name="userLineLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>23</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>原帐号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="mailLineLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>62</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>原邮箱：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mailLineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>170</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLineEdit" name="userLineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>170</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="pwdLineLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>101</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>新密码：</string>
     </property>
    </widget>
    <widget class="MusicUserLineEdit" name="pwdLineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>170</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="pwdLineEditR">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>170</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>6-16个字母、数字和符号组成</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRigh_5">
     <property name="geometry">
      <rect>
       <x>275</x>
       <y>102</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image:url(:/tiny/lb_right)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="MusicCodeAreaWidget" name="verificationCode">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="verificationCodeEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>160</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="verificationCodeLabel">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>164</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>输入验证码：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirmButton_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelButton_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>210</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicUserLineEdit</class>
   <extends>QLineEdit</extends>
   <header>musicuserlineedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicCodeAreaWidget</class>
   <extends>QLabel</extends>
   <header>musiccodeareawidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
